src/scripts/nls/es.js

```diff
@@ -0,0 +1,116 @@
+define({
+    MARK_AS_READ: 'Marcar como (No) Leído',
+    DELETE: 'Eliminar',
+    UPDATE: 'Actualizar',
+    PRINT: 'Imprimir',
+    SEARCH: 'Buscar por...',
+    SEARCH_TIP: 'Buscar artículos. Use \':\' para buscar también dentro del contenido.',
+    NEVER: 'Nunca',
+    EVERY_5_MINUTES: 'Cada 5 minutos',
+    EVERY_15_MINUTES: 'Cada 15 minutos',
+    EVERY_30_MINUTES: 'Cada 30 minutos',
+    EVERY_HOUR: 'Cada hora',
+    EVERY_2_HOURS: 'Cada 2 horas',
+    EVERY_3_HOURS: 'Cada 3 horas',
+    EVERY_5_HOURS: 'Cada 5 horas',
+    EVERY_10_HOURS: 'Cada 10 horas',
+    EVERY_24_HOURS: 'Cada 24 horas',
+    EVERY_WEEK: 'Cada semana',
+    USERNAME: 'Nombre de usuario',
+    PASSWORD: 'Contraseña',
+    CANCEL: 'Cancelar',
+    OK: 'OK',
+    LAYOUT: 'Diseño',
```

Comment:
I'm not sure if there isn't some better word for that

Reply:
Sorry for the delay in replying (had no internet access).

Not really sure what words you would like to use, but I like that you took the time to see if it can be done better.

My thoughts:

PASSWORD: 'Contraseña' -> Pretty standard. Even Gmail uses this in its login page in Spanish.

CANCEL: 'Cancelar' -> This one is pretty standard across most software in Spanish.

OK: 'OK'  -> This one is pretty common too. I guess you could use "Aceptar" instead (Usual in OK/Cancel dialogs --> Aceptar/Cancelar).

LAYOUT: 'Diseño' ->  This one... The alternatives are pretty lame: "Composición", "Distribución". "Trazado". At least in my country (Argentina) "Diseño" is the one that fits better.

So. Maybe the "OK -> Aceptar" wording could be changed.

Thanks again for your work!